socketio_middleware/src/socketio.rs

```diff
@@ -53,9 +53,12 @@ pub async fn broadcast(room_id: &str, event: &str, message: &str) {
                         event.to_string(),
                         message.to_string(),
                     )));
+                    debug!("Found socketid {} in room {}, send message = {}",channel.sid(), room_id, message);
```

Comment:
nit: missing space before `channel.sid()`, also if you could change `send message` to `sending message`.

Reply:
changed